srcpkgs/gopass/template

```diff
@@ -14,14 +14,7 @@ license="MIT"
 homepage="https://www.gopass.pw/"
 changelog="https://raw.githubusercontent.com/gopasspw/gopass/master/CHANGELOG.md"
 distfiles="https://github.com/gopasspw/gopass/archive/v${version}.tar.gz"
-checksum=73b7c5c8367e664d85871fba88f4fa806ef0fb75047c767a72b8516dea6f64b3
-
-post_build() {
-	gopass=$(find $GOPATH/bin -name gopass)
-	for shell in bash fish zsh; do
-		vtargetrun $gopass completion $shell > completion.$shell
-	done
-}
```

Comment:
Why did you remove this? The `vcompletion` calls below will fail because they can't find the file...

Reply:
Generating the completion files during package build time cause trouble for some platforms so we decided to generate them when cutting a release and include them in the source package, see https://github.com/gopasspw/gopass/commit/2fdeb0fd42801026afe178a75ce68d0df29de9f7

But I didn't notice the slightly different naming of these files. Should be fixed now.

Reply:
In that case, please also `vman gopass.1` in `post_install`.

Reply:
Indeed, good idea. Thanks.